.rubocop.yml

```diff
@@ -2,9 +2,11 @@
 require:
   - rubocop-capybara
   - rubocop-factory_bot
+  - rubocop-rspec_rails
+
+plugins:
```

Comment:
gets rid of a deprecation warning that was added in 1.72.x.  i confirmed that this works by making this change first, and continuing to get rubocop rails errors before making the fixes rubocop suggested (see this PR's commit history, where i made those changes as separate commits in order to better show that).